src/editpage/components/response/Goal.vue

```diff
@@ -11,8 +11,13 @@
     <div class="hrfake">
       <span class="goal-id">({{index + 1}}/{{total}})</span>
     </div>
-    <div :class="{opened: opened, 'opened-tick': true}">▶</div>
-    <span class="goal-target" v-html="subGoal"></span>
+    <div v-show="goalOnly">
+      <span class="only-goal-target" v-html="subGoal"></span>
+    </div>
+    <div v-show="showHypotheses">
+      <div :class="{opened: opened, 'opened-tick': true}">▶</div>
+      <span class="goal-target" v-html="subGoal"></span>
+    </div>
```

Comment:
Why wrap everything in extra diffs?
I think we can just add `v-if=showHypotheses` to the triangle div?
Something like:
```vue
<div v-if="showHypotheses" :class="{opened: opened, 'opened-tick': true}">▶</div>
<span :class="{'goal-target': true, 'goal-with-hypotheses': showHypotheses}" v-html="subGoal"></span>
```

And then add the `goal-with-hypotheses` class with the `margin-left: 1em;`